battleship.py

```diff
@@ -118,7 +192,21 @@ def isVertical(ship):
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
+    
+    row=ship[0][0]
+    col=[]
+    for i in range(len(ship)):
+        if(ship[i][0]!=row):
+            return False 
+    for i in range(len(ship)):
+        col.append(ship[i][1])

```

Comment:
You can combine both these loops

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py